content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/ngdmsmartcrop/js/smartcroppreview.js

```diff
@@ -0,0 +1,94 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2023 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+(function($, channel) {
+    "use strict";
+
+    channel.on("change", ".image-v3-dialog-smartcrop-select", function(event, from) {
+        var value = this.value;
+        var $dialog = $(this).closest("coral-dialog.cq-Dialog");
+        var $ele = $dialog.find(".ngdm-smartcrop-thumbnail");
+        var src = $ele.attr("src");
```

Comment:
you have a mix of const and var declaration, all of these could probably be consts, otherwise I think let should be chosen over var.

Reply:
changed.